data/duke.txt

```diff
@@ -0,0 +1,9 @@
+T|1|read book
+D|1|return book|2022-02-02T22:22
```

Comment:
Perhaps this file should be deleted before committing so that the user can test the code to see if it works without a data/duke.txt file being generated at first?